script.js

```diff
@@ -1,21 +1,29 @@
-var firstNumber = document.querySelector(".first");
-var secondNumber = document.querySelector(".second");
-var tirthNumber = document.querySelector(".tirth");
-var btnChange = document.querySelector(".btnChange");
-var fisrtNumberValue, secondNumberValue, tirthNumberValue;
-var bodyElement = document.body;
+// ".first" vine folosit pentru CSS, ".js-first" pentru js
```

Comment:
In rest e bine, dar e prea puțin cod ca sa pot face un review mai serios. 
Spor la treaba.